terraform/modules/hub-key-rotation/s3.tf

```diff
@@ -0,0 +1,8 @@
+resource "aws_s3_bucket" "hub_key_rotation" {
+  bucket = "govukverify-${var.environment}-hub-key-rotation"
```

Comment:
Do we need `${var.environment}` in the bucket name?

Reply:
I would also argue we don't need the `govukverify` either 😬

Reply:
Yep we need it. Bucket names are global, with a region (US, China, rest of the world). The `govukverify` part isn't strictly necessary but is what we do elsewhere and is probably good practices to ensure we're only operating in our own namespace.

Reply:
All right then....